config/initializers/datadog.rb

```diff
@@ -1,23 +1,9 @@
 Datadog.configure do |c|
-  c.tracer env: Rails.env
```

Comment:
Datadog by default will take whatever environment it's in from the agent. This looks like a legacy config.